ci/constantss/host_keys.py

```diff
@@ -0,0 +1,12 @@
+
+OPENSHIFT_NODE = "openshift"
+JENKINS_MASTER_NODE = "jenkins_master"
+JENKINS_SLAVE_NODE = "jenkins_slave"
+SCANNER_NODE = "scanner"
+CONTROLLER_NODE = "controller"
+
+HOSTS_HOST = "host"
+HOSTS_PRIVATE_KEY = "private_key"
+HOSTS_REMOTE_USER = "remote_user"
+HOSTS_USER = "user"
+HOSTS_INVENTORY = "inventory_path"
```

Comment:
where are we using these?

Reply:
Hmm how did those even get there o_O that is supposed to be from ci rewrite? Will remove the same.